### DISCUSSION TITLE
Mouse Hover event for points, edges, faces

### DISCUSSION THREAD

**Discussion by royvelich** [pygfx/pygfx#764] [Q&A] [answered]
Hi,
Is there a native way to handle and capture mouse hover/click events for points/edges/faces? Any example for it?

Thanks :-)

**Comment by Korijn** [comment, marked as answer]
See https://github.com/pygfx/pygfx/blob/main/examples/feature_demo/picking_mesh.py

There are more examples in this folder that showcase picking

**Reply by royvelich** [reply]
Thanks

**Comment by kushalkolar** [comment]
All the events are defined here: https://github.com/pygfx/pygfx/blob/main/pygfx%2Fobjects%2F_events.py#L13

I've been thinking of doing a PR with a table that describes these so it's easy to find in the docs.

**Reply by royvelich** [reply]
Thanks
